.cirrus.yml

```diff
@@ -124,6 +124,27 @@ task:
     - name: web_tests-7_last-linux # last Web shard must end with _last
       << : *WEB_SHARD_TEMPLATE
 
+    - name: web_integration_test_linux
+      compile_host_script: |
+        cd $ENGINE_PATH/src
+        ./flutter/tools/gn --unoptimized --full-dart-sdk
+        ninja -C out/host_debug_unopt
+      fetch_framework_script: |
+        cd $ENGINE_PATH/src/flutter/tools
+        ./clone_flutter.sh
+        cd $FRAMEWORK_PATH/flutter
+        bin/flutter update-packages --local-engine=host_debug_unopt
+      script:
+        - git clone https://github.com/flutter/web_installers.git
+        - cd web_installers/packages/web_drivers/
+        - $ENGINE_PATH/src/third_party/dart/tools/sdks/dart-sdk/bin/pub get
+        - $ENGINE_PATH/src/third_party/dart/tools/sdks/dart-sdk/bin/dart lib/web_driver_installer.dart chromedriver --install-only
+        - ./chromedriver/chromedriver --port=4444 &
+        - cd $ENGINE_PATH/src/flutter/e2etests/web/text_editing
+        - $FRAMEWORK_PATH/flutter/bin/flutter config --local-engine=host_debug_unopt --no-analytics --enable-web
+        - $FRAMEWORK_PATH/flutter/bin/flutter pub get --local-engine=host_debug_unopt
```

Comment:
Is `--local-engine=host_debug_unopt` necessary in these two commands?

Reply:
yes. otherwise it gives an error. I tried without them initially.

Reply:
@jonahwilliams @yjbanov any idea why this is the case? I would assume `config` and `pub get` have nothing to do with local engine, no?

Reply:
The goal for running these tests is to make sure everything we build in the engine is functional. Double-checking `pub get` (which we also build here) can't hurt.

Reply:
A local engine will override all of the artifacts, even pub. If that is what you want to test, makes sense to do it consistently

Reply:
Actually, now that I think of it, having to specify `--local-engine` everywhere seems error-prone. Maybe let's switch to the `LOCAL_ENGINE` environment variable to make sure we don't miss it?

@jonahwilliams Is `LOCAL_ENGINE` a drop-in replacement for `--local-engine`?

Reply:
That's good to know. I had no idea `--local-engine` had an effect on `flutter pub get`.

Reply:
I don't think LOCAL_ENGINE does anything. We use this to communicate with bash scripts but the tool itself doesn't use it

Reply:
Thanks all for the questions and information. Keeping --local-engine=host_debug_unopt as it is now.